src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -213,11 +213,11 @@ public void withRMarkdown(String progressCaption, String userAction,
       deps.add(Dependency.cranPackage("htmltools", "0.3.5"));
       deps.add(Dependency.cranPackage("caTools", "1.14"));
       deps.add(Dependency.cranPackage("bitops", "1.0-6"));
-      deps.add(Dependency.cranPackage("knitr", "1.14", true));
+      deps.add(Dependency.cranPackage("knitr", "1.14"));
```

Comment:
Based on comments on that commit it's so that we can install knitr even if CRAN hasn't built the binaries. Maybe not a problem for 1.14?